### PR TITLE
Feature/fit tracker controllers

### PR DESCRIPTION
**Pull request by KyletheWolff**
#26 The views are connected to controllers now

### CODE REVIEW

File: src/com/kandbnutrition/controller/CustomerFitTrackerCell.java [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+	Line leftLine, rightLine;
+	
+	@FXML
+	TextField exerciseTextField,  weightTextField, repsTextField, setsTextField;
+	
+	private StringValues stringValues;
+	
+	public CustomerFitTrackerCell() {
+		
+		stringValues = new StringValues();
```
Comment (BrandonVanderMey) [inline src/com/kandbnutrition/controller/CustomerFitTrackerCell.java:32]: I don't like how we have to create a new object to retrieve string values.  We should access this statically.

Comment (KyletheWolff) [inline src/com/kandbnutrition/controller/CustomerFitTrackerCell.java:32]: We could make another enum? Static is what I am trying not to do.

**Comment by BrandonVanderMey** [comment]
What is wrong with static?  Is there any reason other that it feels hacky?

File: src/com/kandbnutrition/controller/CustomerFitTrackerCell.java (hunk 1 above)
Comment (BrandonVanderMey) [inline src/com/kandbnutrition/controller/CustomerFitTrackerCell.java:32]: What is wrong with static? Is there any reason other that it feels hacky?

Comment (KyletheWolff) [inline src/com/kandbnutrition/controller/CustomerFitTrackerCell.java:32]: Thread saftey, code implicity

Comment (BrandonVanderMey) [inline src/com/kandbnutrition/controller/CustomerFitTrackerCell.java:32]: But nothing is being changed in the class.  The string values stay the same no matter what thread access them first

Comment (KyletheWolff) [inline src/com/kandbnutrition/controller/CustomerFitTrackerCell.java:32]: Alright well we will need a story for that. It should now hold up this
merge.

**Pull request merged**